compiler/src/glob_options.ml

```diff
@@ -141,8 +141,8 @@ let print_strings = function
   | Compiler.RemoveGlobal                -> "rmglobals", "remove globals variables"
   | Compiler.MakeRefArguments            -> "makeref"  , "add assignments before and after call to ensure that arguments and results are ref ptr"
   | Compiler.LowerInstruction            -> "lowering" , "lowering of instructions"
-  | Compiler.SLHLowering                  -> "slhlowering" , "selective load hardening lowering of instructions"
   | Compiler.PropagateInline             -> "propagate", "propagate inline variables"
+  | Compiler.SLHLowering                  -> "slhlowering" , "selective load hardening lowering of instructions"
```

Comment:
Maybe, "lowering of selective load hardening instructions" will be better. 
Not really related to this PR. but since you touch it.

Reply:
Done